Libplanet/Blockchain/BlockChain.cs

```diff
@@ -960,8 +960,6 @@ public long GetNextTxNonce(Address address)
             finally
             {
                 TipChanged -= WatchTip;
-                cancellationTokenSource.Dispose();
-                cts.Dispose();
```

Comment:
It wasn't disposed of here?

Reply:
it can be bypassed if exceptions were raised before `try` block.